src/lpwFuncs.cpp

```diff
@@ -273,6 +250,42 @@ void userActivity() { // Call of this function can be found in line 181, buttons
   inactivityTime=0;   // reset the inactivity counter!
 }
 
+void initDormant() {
+  delay(2000);
+
+  #ifdef AUDIO_SIGNAL_PIN
+    pwm_set_enabled(pwm_gpio_to_slice_num(AUDIO_SIGNAL_PIN), false);
+  #endif
+  #ifdef TONE_PIN
+    pwm_set_enabled(pwm_gpio_to_slice_num(TONE_PIN), false);
+  #endif
+  alarm_pool_destroy(app_alarm_pool);
+  #ifdef IR_LED_PIN
+    stop_IR_command();
+  #endif
+
+  MouseBLE.end(); delay(500); KeyboardBLE.end(); delay(500);
+  #ifdef FABIJOYSTICK_ENABLED
+    JoystickBLE.end();
+  #endif
+
+  cyw43_arch_deinit();
+
+  displayMessage((char*) "ByeBye");
+  pauseDisplayUpdates(1);
+  delay(2000);
+  displayClear();
+
+  if (Wire.available()) {
+    Wire.flush(); delay(10); Wire.endTransmission(); delay(10); Wire.end(); delay(100); 
+  }
+  if (Wire1.available()) {
+    Wire1.flush(); delay(10); Wire1.endTransmission(); delay(10); Wire1.end(); delay(100);
+  }
```

Comment:
maybe put that in a dedicated function, and into a module than handles I2C init / testing / deactivation?

Did you find that shutting down Wire and Wire1 significantly reduces power consumption during sleep?